src/web/client/dal/remoteFetchProvider.ts

```diff
@@ -119,6 +111,14 @@ async function fetchFromDataverseAndCreateFiles(
                 throw new Error(ERRORS.EMPTY_RESPONSE);
             }
 
+            WebExtensionContext.telemetry.sendAPISuccessTelemetry(
+                requestUrl,
+                entityName,
+                Constants.httpMethod.GET,
+                new Date().getTime() - requestSentAtTime,
+                fetchFromDataverseAndCreateFiles.name
```

Comment:
@tyaginidhi will this log the entire file name or just the file extension?

Reply:
none - file extension is logged in commonUtil -> isWebfileContentLoadNeeded